src/renderer/views/Trending/Trending.js

```diff
@@ -202,7 +195,7 @@ export default Vue.extend({
           }
         })
 
-        if (!process.env.IS_ELECTRON || (this.backendPreference === 'invidious' && this.backendFallback)) {
+        if (process.env.IS_ELECTRON && (this.backendPreference === 'invidious' && this.backendFallback)) {
```

Comment:
This check should also be in a couple of other places that fall-backs occur.
- line 159 of src/renderer/views/Playlist/Playlist.js
- line 530 of src/renderer/views/Channel/Channel.js
- line 769 of src/renderer/views/Watch/Watch.js
- line 285 of src/renderer/views/Search/Search.js
- line 360 of src/renderer/components/watch-video-playlist/watch-video-playlist.js
- line 259 of src/renderer/components/top-nav/top-nav.js
- line 92 of src/renderer/components/watch-video-live-chat/watch-video-live-chat.js

An alternative to adding the check in all of these places might be to force `this.backendFallback` to always be `false` if `process.env.IS_ELECTRON` is `false`.